topostats/plottingfuncs.py

```diff
@@ -89,7 +108,17 @@ def plot_and_save(
 
         if save:
             if image_set == "all" or core_set:
-                plt.savefig(output_dir / filename)
+                if not axes and not colorbar:
+                    plt.title('')
```

Comment:
Have you linted this code with [Black](https://github.com/psf/black)? I think it would have replaced single quotes with double quotes in most instances if so.

Reply:
My black in VS code doesn't seem to do quotes. I think I need to look at you pylint docs again.

I've done this manually for now